package.json

```diff
@@ -99,7 +99,7 @@
     "css-minimizer-webpack-plugin": "7.0.2",
     "date-fns": "4.1.0",
     "electron": "35.1.2",
-    "electron-builder": "26.0.12",
+    "electron-builder": "25.1.8",
```

Comment:
But it needs to be downgraded then?

Reply:
At the moment, yes. The newer electron-builder throws some errors I haven't had time to debug yet